src/PinchZoom/component.tsx

```diff
@@ -951,21 +959,24 @@ class PinchZoom extends React.Component<RequiredProps & DefaultProps> {
     const props = containerProps || {};
 
     return (
-      <div
-        {...props}
-        ref={this._containerRef}
-        className={classnames(styleRoot, props.className)}
-      >
-        {React.cloneElement(child, {
-          className: classnames(styleChild, child.props.className)
-        })}
-      </div>
+      <>
+        <style>{styles}</style>
```

Comment:
Maybe better to inject CSS on the side App?

```js
import 'react-quick-pinch-zoom/styles.css';

function App(){

  return (
    <QuickPinchZoom onUpdate={onUpdate}>
      <img ref={imgRef} src={IMG_URL} />
    </QuickPinchZoom>
  );
}
```
```

Reply:
Sorry I don't understand what you mean.

Reply:
Look at example https://nextjs.org/docs/basic-features/built-in-css-support#import-styles-from-node_modules

Next js have the ability to add custom CSS on the page 

instead of creating `<style>` tag for each components

Reply:
I know of that nextjs feature, but taking advantage of it would require to create a separate `.css` file that plugin users would have to take care of loading/requiring when using the plugin. That would be a breaking change then.

The previous implementation was referencing the DOM outside of the component to inject the styles in the head and that was breaking SSR.

As the styles are needed for the plugin to function (or so I assume) the React way to add them would be to inline them via `style` prop IMO, but I did not know if that would interfere with the calculated styles.

So the cleanest solution I found, as it will destroy de style node on unmount it to just include it in the render method as I did.

This solution will also not be a breaking change for existing users of this plugin.

Reply:
How about injecting styles only for a browser env?

And for SSR env create a note in `REDEME.md` that requred to inject css ot their side

Reply:
I am not sure how to approach that. 

What would be the benefit of that approach? And what is the downside you see to the approach I propose?